snuba/views.py

```diff
@@ -328,7 +332,10 @@ def parse_and_run_query(dataset, request: Request, timer) -> QueryResult:
         'sample': request.query.get_sample(),
     }
 
-    return raw_query(request, query, clickhouse_ro, timer, stats)
+    with sentry_sdk.start_span(description=query.format_sql(), op="db") as span:
+        span.set_tag("dataset", dataset.__class__.__name__)
```

Comment:
```suggestion
        span.set_tag("dataset", type(dataset).__name__)
```

Not a big deal, but `type` is generally preferred — I forget why, might have been a bigger issue in Python 2 than 3.